### DISCUSSION TITLE
Project submitted but azure credit expired!

### DISCUSSION THREAD

**Discussion by Riyashaw17** [github/india#799] [Students]
Hello, My name is Riya Shaw and I have a major problem now. As the title says, I have successfully uploaded my project on 26th February and my project was in total  working condition. But now my azure credits are totally exhausted and hence my web page is not working.

I would be highly obliged if You could please consider my situation and provide me with a solution as I do not want my hardwork and time to go in vain like this. 

I have already provided the necessary screenshots related to my project in my GitHub repository Readme file. You can refer to it.

Thank You
Riya Shaw

**Comment by ey-mentor1** [comment]
Hello,

If there are sufficient relevant screenshots and all the mandatory fields are present, then there would be no rejection.
And, if there are some unsatisfactory details,  in that case you have to reshoot or take screenshots again.

Please create Azure free account, incase you have exhausted all credits given by FRT.

Thank you

**Comment by Riyashaw17** [comment]
Thank You for your response. I will be waiting for the evaluation result.
